CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ option( k-computer "Enable K computer." OFF )
 set( disable-timing OFF CACHE BOOL "Disable measurements via Stopwatches. [default OFF]" )
 set( disable-counts ON CACHE BOOL "Disable measurements of call counts. [default ON]" )
 
+set( target-bits-split "standard" CACHE STRING "Split of the 64bits of the Target class. 'standard' is recommended for most users. If running on more than 262144 MPI processes or more than 512 threads, change to 'hpc'. [default standard]" )
```

Comment:
The description can perhaps be more specific: instead of "64bits of the Target class" I would suggest "64-bit target neuron identifier type".